### PR TITLE
Fix Shimmer UI in OpenPRs and StalePRs page (closes #153)

### LINKED ISSUE #153: Fix shimmer UI in OpenPRs and StalePRs page

### PR DESCRIPTION
**Pull request by Vividh25**
Issue: 
-  The loading screen shimmer UI was going out of bounds #153 

Fix: 
-  Made changes to the scss file of the shimmer component. 

Before: 

before

After Fix: 

https://user-images.githubusercontent.com/61474917/121961237-c76b3080-cd84-11eb-8460-e6b0c6a0c520.mp4

Making the Shimmer effect look like the corresponding cards: 

https://user-images.githubusercontent.com/61474917/122036042-bd7f1700-cdf0-11eb-91a8-0c694cd24738.mp4

### CODE REVIEW

File: src/components/Loaders/cardShimmer/index.js [hunk 1]
```diff
@@ -1,15 +1,21 @@
-import React from 'react';
-import classNames from './cardShimmer.module.scss';
+import React from "react";
```
Comment (sebuelias) [inline src/components/Loaders/cardShimmer/index.js:1]: single-quote and double-quote confusions, it is better to check and stick with the projects style guide. And not use your personal preference.
Check the eslint config for the project or you can refer other files

Comment (Vividh25) [reply src/components/Loaders/cardShimmer/index.js:1]: Okay, I'll fix it.

Comment (Vividh25) [reply src/components/Loaders/cardShimmer/index.js]: I've made the changes please check.

File: src/components/Loaders/cardShimmer/index.js [hunk 2]
```diff
@@ -2,14 +2,20 @@ import React from 'react';
...
+      />
+      <div
+        className={`${classNames.create} ${classNames.br} ${classNames.animate}`}
+      />
+      <div
+        className={`${classNames.comment} ${classNames.br} ${classNames.animate}`}
+      />
+      <div
+        className={`${classNames.comment} ${classNames.br} ${classNames.animate}`}
+      />
```
Comment (whyDontI) [inline src/components/Loaders/cardShimmer/index.js:18]: This is just an indentation change, let's revert it back

**Review by whyDontI — CHANGES_REQUESTED** [review]
A couple of things needs to be changed,
- It'll be great if you can nest the CSS code in `sass` style
- Make the shimmer responsive

Screenshot 2021-06-25 at 1 55 29 AM

File: src/components/Loaders/cardShimmer/index.js (hunk 2 above)
Comment (Vividh25) [reply src/components/Loaders/cardShimmer/index.js:18]: I'll fix the issues, but I think the shimmer is already responsive, see this: 

https://user-images.githubusercontent.com/61474917/123444884-0145fd80-d5f5-11eb-924c-4a14f9c1a8e6.mp4

This is how it looks in full screen mode

Comment (Vividh25) [reply src/components/Loaders/cardShimmer/index.js:18]: Also, do you want me to nest all the css in the in `cardShimmer.module.scss` into `sass` style ?

Comment (whyDontI) [reply src/components/Loaders/cardShimmer/index.js]: - Sure please nest the `scss`
- Can you make the shimmer occupy full width in mobile?

RDSStatusShimmer

File: src/components/Loaders/cardShimmer/cardShimmer.module.scss [hunk 3]
```diff
@@ -46,12 +52,6 @@
   }
 }
 
-.animate {
-  animation: shimmer 2s infinite linear;
-  background: linear-gradient(to right, #eff1f3 4%, #e2e2e2 25%, #eff1f3 36%);
-  background-size: 1000px 100%;
-}
-
 @keyframes shimmer {
```
Comment (correaswebert) [inline src/components/Loaders/cardShimmer/cardShimmer.module.scss:55]: I guess @whyDontI  meant to say integrate this as well into the `.wrapper` block, just like `fullview` 👍🏾

File: src/components/Loaders/cardShimmer/index.js (hunk 2 above)
Comment (Vividh25) [reply src/components/Loaders/cardShimmer/index.js]: This is how it looks now: 

https://user-images.githubusercontent.com/61474917/127209605-176c5cd7-d31a-4d5b-a8e8-03dc8096d880.mp4

**Comment by whyDontI** [comment]
Great work @Vividh25 🥳
Closing this PR, picking changes from here to apply in another issue. 👍🏽

**Pull request closed without merging**